### PR TITLE
Adds fun back to the game

### PR DESCRIPTION
**Pull request by Robustin**
Revenants now take more dead mobs and more time to appear.

1) Lavaland roles meant revenants were spawning every fucking round holy shit

2) Revenants were basically unkillable fuckbuddies for every antag on the server

3) FACT: Revenants have never fucking made a round better. We call them Grief Ghosts because all they do is put security players on suicide watch because holy shit they're a fucking nightmare to deal with.

P.S. This isn't an ided post, I just watched decent god-fearing family-having security players get absolutely reamed because they were getting EMP'd or Blighted every 20 fucking seconds.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
inb4 title isnt changed and the bias comes through

**Comment by PKPenguin321** [comment]
thank god

**Comment by KazeEspada** [comment]
To be honest... im not sure what a revenant needs all those spells for

File: code/game/gamemodes/miniantags/revenant/revenant_spawn_event.dm [hunk 1]
```diff
@@ -1,11 +1,11 @@
-#define REVENANT_SPAWN_THRESHOLD 10
+#define REVENANT_SPAWN_THRESHOLD 20
 
 /datum/round_event_control/revenant
 	name = "Spawn Revenant"
 	typepath = /datum/round_event/revenant
 	weight = 7
 	max_occurrences = 1
-	earliest_start = 6000 //Meant to mix things up early-game.
+	earliest_start = 12000 //Meant to mix things up early-game.
```
Comment (Man-Shroom-) [inline code/game/gamemodes/miniantags/revenant/revenant_spawn_event.dm]: what is 12000

**Comment by AlexanderUlanH** [comment]
:-1: Rebalance revenant powers to require more crew interaction and vulnerability / make them less capable of persistently fucking with security if that's the problem.

**Comment by ChangelingRain** [comment]
If a griffghost has both EMP and blight and can spam them you have already lost because frankly that's 350 essence for both spells and 40/50 a cast, respectively.

**Comment by Robustin** [comment]
This doesn't even remove the goddamn grief ghost, just makes sure its not an every-other-round occurrence that is exclusively designed to fuck security and non-antag players in the ass.

File: code/game/gamemodes/miniantags/revenant/revenant_spawn_event.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/game/gamemodes/miniantags/revenant/revenant_spawn_event.dm]: 20 minutes i think

**Comment by PKPenguin321** [comment]
yeah if anything this just makes griefghosts more in-line with their pre-lavaland spawn rates
could almost be seen as a fix

**Comment by Bawhoppen** [comment]
>title isn't being changed

reeeeeeeeeeeeeeeeeeeeeeeeeeeeee

File: code/game/gamemodes/miniantags/revenant/revenant_spawn_event.dm (hunk 1 above)
Comment (Robustin) [inline code/game/gamemodes/miniantags/revenant/revenant_spawn_event.dm]: 20 minutes instead of 10.

**Comment by Zilenan91** [comment]
Why are revenants spawning every round from lavaland?

**Comment by Robustin** [comment]
Thump up users = God fearing faithful men and women who hold the line every day and make SS13 a better place. They love their families and are just a few days from retirement, all they want is just one shift in the brig that isn't an absolute hell.

Thumbs down users = Coder memes who've never experienced the migraine-inducing hellscape caused when antag and revenants team up, which is practically 50% of rounds now. 

@Zilenan91

Because Revenants require 10 dead players to trigger. Before lavaland it actually took some time to hit that threshold. Now in lavaland with miners/ghost-roles/NPC's murderizing each other its basically guaranteed to hit the revenant minimum every round within 10 minutes. Only by the holy grace of RNGesus will a round be spare from the shittery of griffghosts.

**Comment by AlexanderUlanH** [comment]
The problem isn't that revenants happen too often. The problem is that revenants' abilities make them untouchable griff machines.

**Comment by AdamElTablawy** [comment]
:-1: i like rolling revenant every round and being a gigantic asshole


but aside from that yes this is probably a good idea

**Comment by TechnoAlchemisto** [comment]
obnoxious little creatures that give me a headache but lack the lethality to put me out of my misery

**Comment by coiax** [comment]
If everyone agrees that revenants add nothing to the game, why are they still in?

**Comment by idontwannasignup** [comment]
That's taking things out of context, they're a fun role, but need to be either weaker, slower, or more visible.

**Comment by Robustin** [comment]
>Nerf revenant
I'm already facing stiff opposition just to get it to appear as often as it did 2 months ago.
>Revenant is fun
It's really, really not. Invisible uninteractive grieflords just that make your round a nightmare are not fun in any way, shape, or form.

**Comment by yackemflam** [comment]
Get the chaplain to beat the revenants ass

**Comment by Supermichael777** [comment]
I don't know how Griff ghost has stayed this long. people use it to be unambiguously assholes so its impassible for me to do anything fun with it because people reflexively kill.

**Comment by Incomptinence** [comment]
Oh that explains people having break downs over the poor little revenant I guess.

**Comment by KorPhaeron** [comment]
You're misreading the code, it's not dead players, it's

>for(var/mob/M in dead_mob_list)

So any dead mob of any kind counts.

**Comment by Iamgoofball** [comment]
@KorPhaeron 
so this is why every time I slaughter the 50K kittens in my office as the CMO the revenants show up

**Comment by KorPhaeron** [comment]
Revenge of catsplosion

**Comment by Robustin** [comment]
Yea I saw that, meant to correct it, still no shortage of dead mobs now that mining is so sought-after.

Yay for miners, but there are still side-effects that need to be addressed.

**Comment by Bawhoppen** [comment]
>title was never changed

REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Pull request merged**